### PR TITLE
Removed colons from labels that don't need them

### LINKED ISSUE #4639: Form label style is inconsistent
The Wikidata dialogs and the create/import project panels append (:) to the labels which are not consistent with the rest of the application.

In general for web applications, colons are no longer used in modern web design.

> Don’t add a colon (:) at the end of a field label to imply that the label text applies to the field it accompanies. The design of the component already communicates the relationship between the label and the input field.
Reference: https://spectrum.adobe.com/page/field-label/

### PR DESCRIPTION
**Pull request by elebitzero**
Fixes #4639

Changes proposed in this pull request:
- Removed : and ... from labels that don't need them.

image
image
image

### CODE REVIEW

File: extensions/database/module/langs/translation-fr.json [hunk 1]
```diff
@@ -15,15 +15,15 @@
...
-    "database-source/databaseTypeLabel": "Type :",
-    "database-source/databaseHostLabel": "Hôte :",
-    "database-source/databasePortLabel": "Port :",
-    "database-source/databaseUserLabel": "Utilisateur :",
-    "database-source/databasePasswordLabel": "Mot de passe :",
-    "database-source/databaseNameLabel": "Base de données :",
-    "database-source/databaseFileNameLabel": "Fichier de données :",
-    "database-source/databaseSchemaLabel": "Schéma :",
+    "database-source/connectionNameLabel": "Nom ",
+    "database-source/databaseTypeLabel": "Type ",
```
Comment (wetneb) [inline extensions/database/module/langs/translation-fr.json:19]: I guess we probably also want to remove the trailing whitespace in those strings.

File: extensions/wikidata/module/langs/translation-fr.json [hunk 2]
```diff
@@ -58,8 +58,8 @@
     "wikibase-preview/new-id": "nouvel élément",
     "wikibase-account/dialog-header": "compte $1",
     "wikibase-account/explain-log-in": "Se connecter à <a href=\"$1\" target=\"_blank\">$2</a> vous permet de publier vos données sur Wikidata depuis OpenRefine.",
-    "wikibase-account/username-label": "Nom d'utilisateur :",
-    "wikibase-account/password-label": "Mot de passe :",
+    "wikibase-account/username-label": "Nom d'utilisateur ",
```
Comment (wetneb) [inline extensions/wikidata/module/langs/translation-fr.json:61]: Same here.

File: main/webapp/modules/core/langs/translation-fr.json [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
     "name": "Français",
     "core-index/help": "Aide",
-    "core-index/contributors": "Contributeurs :",
+    "core-index/contributors": "Contributeurs ",
     "core-index/new-proj-name": "Nom du nouveau projet :",
     "core-index/download-now": "Télécharger $1 maintenant.",
     "core-index/delete-key": "Supprimer la préférence",
-    "core-index/id": "ID du projet :",
-    "core-index/subject": "Sujet :",
+    "core-index/id": "ID du projet ",
```
Comment (wetneb) [inline main/webapp/modules/core/langs/translation-fr.json:8]: And here.

Comment (elebitzero) [reply main/webapp/modules/core/langs/translation-fr.json]: Removed the trailing spaces and updated the tests.

**Review by wetneb — APPROVED** [review]

**Pull request merged**